src/TLuaInterpreter.cpp

```diff
@@ -2361,6 +2361,25 @@ int TLuaInterpreter::disableKey( lua_State *L )
     return 1;
 }
 
+int TLuaInterpreter::killKey( lua_State *L )
+{
+    string luaSendText="";
+    if( ! lua_isstring( L, 1 ) )
+    {
+        lua_pushstring( L, "killKey: wrong argument type" );
```

Comment:
To support the __I18n__ work for Mudlet 4.0 you will see later code in this class containing more informative and translatable error code such as:

    lua_pushstring( L, tr( "killKey: bad argument #1 type (key name as string expected, got %1!)" )
                               .arg( luaL_typename( L, 1 ) )
                               .toUtf8().constData() );